plugins/omelasticsearch/omelasticsearch.c

```diff
@@ -570,6 +573,10 @@ computeMessageSize(const wrkrInstanceData_t *const pWrkrData,
 	uchar *pipelineName;
 
 	getIndexTypeAndParent(pWrkrData->pData, tpls, &searchIndex, &searchType, &parent, &bulkId, &pipelineName);
+DBGPRINTF("computeMessageSize: message %p, searchIndex %p, searchType%p\n", message, searchIndex, searchType);
```

Comment:
Are those debug messages sill required?

Reply:
For the time being, yes (we are still looking at some issue).